README.md

```diff
@@ -195,6 +195,8 @@ provided in the table below:
 |  :activity-delete/finish | API | Entry delete request finished, refresh the board data. |
 |  :activity-get/finish | API | Activity (secure or not) loaded from API. |
 |  :activity-modal-edit | UI | Activate editing in modal. |
+|  :activity-modal-fade-in | UI | Show the activity modal without reloading the route. |
```

Comment:
I'm not sure why this was moved to a dispatch when it is an action creator that ends up using a set route.  I don't see a need for this to be part of the application state since I can't see where any other component uses it.

I suggest moving this code to a new file `/actions/[activity|entry|post].cljs`  Not sure what the naming convention is?